packages/lib/src/interactions/DefaultInteractions.tsx

```diff
@@ -1,58 +1,80 @@
-import { merge } from 'lodash';
-
 import Pan from '../interactions/Pan';
+import type { PanProps } from '../interactions/Pan';
 import SelectToZoom from '../interactions/SelectToZoom';
+import type { SelectToZoomProps } from '../interactions/SelectToZoom';
 import XAxisZoom from '../interactions/XAxisZoom';
+import type { XAxisZoomProps } from '../interactions/XAxisZoom';
 import YAxisZoom from '../interactions/YAxisZoom';
+import type { YAxisZoomProps } from '../interactions/YAxisZoom';
 import Zoom from '../interactions/Zoom';
+import type { ZoomProps } from '../interactions/Zoom';
 import AxialSelectToZoom from './AxialSelectToZoom';
-import type { Interactions } from './models';
-import { getDefaultInteractions } from './utils';
+import type { AxialSelectToZoomProps } from './AxialSelectToZoom';
+
+export interface DefaultInteractionsConfig {
+  pan?: PanProps | false;
+  zoom?: ZoomProps | false;
+  xAxisZoom?: XAxisZoomProps | false;
+  yAxisZoom?: YAxisZoomProps | false;
+  selectToZoom?: Omit<SelectToZoomProps, 'keepRatio'> | false;
+  xSelectToZoom?: Omit<AxialSelectToZoomProps, 'axis'> | false;
+  ySelectToZoom?: Omit<AxialSelectToZoomProps, 'axis'> | false;
+  withAxialZoom?: boolean; // shortcut to turn off all axial zoom interactions
```

Comment:
`withAxialZoom` is just a convenience prop to disable axial zoom completely. This is useful for the `ScatterVis` currently (see comment).

Reply:
It feels a bit redundant with `keepRatio` that has the same effect. I would not add it.